subsys/bluetooth/host/smp.c

```diff
@@ -1876,6 +1880,10 @@ static void smp_pairing_complete(struct bt_smp *smp, uint8_t status)
 
 		if (bt_auth && bt_auth->pairing_complete) {
 			bt_auth->pairing_complete(conn, bond_flag);
+#if defined(CONFIG_BT_SETTINGS)
+			/* Pairing has completed, so persist the CCC and CF data. */
+			k_work_submit(&smp->work_settings_ccc_and_cf_save);
+#endif /* CONFIG_BT_SETTINGS */
```

Comment:
Why is this done here, and not the same place where the `identity_resolved` callback is called?

Reply:
Not used anymore.